backend/devices/bitbox/device.go

```diff
@@ -859,7 +859,7 @@ func (dbb *Device) Random(typ string) (string, error) {
 	dbb.log.WithField("random", rand).Debug("Generated random")
 	if len(rand) != 32 {
 		dbb.log.WithField("random-length", len(rand)).Error("Unexpected length: expected 32 bytes")
-		return "", fmt.Errorf("unexpected length, expected 32, got %d", len(rand))
+		return "", fmt.Errorf("unexpected length, expected 32, got %d", len(rand)) //nolint:goerr113
```

Comment:
what does the linter want here instead?

what about disabling the goerr113 linter, seems there is no way to do it right except when we get rid of `errp` or extend it to be 1.13 compatible? we also need to be careful to keep Go1.12 working for bitbox02-api-go until GopherJS supports newer.

Reply:
oh, very good point about gopherjs. I completely forgot tbh.
reverted, disabled goerr113 and added a comment in golangci.yml.

which reminds me, this line should read go1.12: https://github.com/digitalbitbox/bitbox02-api-go/blob/8e5a24b5/.travis.yml#L12, and we probably need to add a build on go1.12 here too.

Reply:
or better yet, actually do the js compilation as part of CI.

Reply:
ah, js compilation already happens but in api-js repo: https://github.com/digitalbitbox/bitbox02-api-js/blob/d9b4718b/.github/workflows/ci.yml#L16-L20

but another good point indeed. best to fail right away in api-go if it's already known we won't be able to compile in api-js.